hnn_core/viz.py

```diff
@@ -491,6 +486,10 @@ def plot_spikes_raster(cell_response, trial_idx=None, ax=None, show=True):
         The CellResponse object from net.cell_response
     trial_idx : int | list of int | None
         Index of trials to be plotted. If None, all trials plotted
+    tmin : float or None
+        Start time of plot in milliseconds. If None, plot entire simulation.
+    tmax : float or None
+        End time of plot in milliseconds. If None, plot entire simulation.
```

Comment:
Why was this added? I agree that time bounds should be included somehow when generating a raster plot, but I believe we can just use `cell_response.times`.

Reply:
@rythorpe for reference a lot of these were just copied from the previous PR. @tianqi-cheng you can remove tmin/tmax

Reply:
Ah okay. Since the diff is different here than in #604, I guess some of commits didn't get copied over. Perhaps we should cherry-pick the remaining commits so that we don't have to rehash some of our previous discussion points?

Reply:
We did cherry-pick, but PR had open comments that were not addressed. I just updated the codes and deleted the tmin and tmax and other related codes.